Documentation~/menu-editors.md

```diff
@@ -2,17 +2,15 @@
 
 This sub-menu provides access to tools and editor windows.
 
-![Tools > ProBuilder > Editors menu](images/menu-editors.png)
-
-## New Bezier Shape
+## Create Bezier Shape
```

Comment:
@JonMUnity there's a comment from May on ticket 5820 that says to change "new" to "create" for three editors, but in my version of U6 + PB6 it still says "new".

Reply:
Let's go with what the UI says in that case. Sorry.

Reply:
```suggestion
## New Bezier Shape
```